src/main/java/produto/TipoTamanho.java

```diff
@@ -1,8 +1,6 @@
 package produto;
 
 public enum TipoTamanho {
-    //IMPLEMENTE A LOGICA DO ENUM
-    //TODO
-    public final double multiplicador;
+    P,M,G;
 
 }
```

Comment:
public enum TipoTamanho {
        P (1.0), M(1.3), G(1.5);

        public final double multiplicador;

        TipoTamanho(final double multiplicador) {
            this.multiplicador = multiplicador;
        }
    }

Reply:
Muito obrigada! Clareou bem como implementar dessa forma agora

Reply:
Resolvido no commit 7c915cf8d11e10a46fd27d1c56aab880b7bca9ba